sources/libs/brutal/text/cp437.c

```diff
@@ -0,0 +1,177 @@
+#include <brutal/text/cp437.h>
+
+uint8_t cp437_from_rune(Rune rune)
+{
+    if (rune >= U'\x20' && rune < U'\x7f')
+    {
+        return (uint8_t)rune;
+    }
+
+    // clang-format off
+    switch (rune)
+    {
+        case U'☺': return 0x1;
```

Comment:
why not an array ?

Reply:
An array would be massive with a lot of holes